### PR TITLE
chore: resolve i18 lint errors 

### PR DESCRIPTION
**Pull request by zwsaqx**
# Description

A new lint rules was added to require translation tags and the lint errors from that were resolved.

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/35391220/todos/6944429305)

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Running 'nx run shared-ui:lint' should return no errors
- [ ] Running 'nx run nexus-admin:lint' should return no errors
- [ ] Running 'nx run journeys:lint' should return no errors
- [ ] Running 'nx run docs:lint' should return no errors
- [ ] Running 'nx run watch:lint' should return no errors
- [ ] Running 'nx run journeys-admin:lint' should return no errors

(you can also run all these tests at once with: 'nx run-many -t lint')

### CODE REVIEW

File: apps/journeys-admin/src/libs/TestEditorState/TestEditorState.tsx [hunk 1]
```diff
@@ -14,18 +15,41 @@ export function TestEditorState({
   renderChildren
 }: TestEditorStateProps): ReactElement {
   const { state } = useEditor()
+  const { t } = useTranslation('apps-journeys-admin')
```
Comment (csiyang) [inline apps/journeys-admin/src/libs/TestEditorState/TestEditorState.tsx:18]: why is this change needed?

File: apps/journeys-admin/src/components/Editor/VideoBlockEditor/Source/SourceFromYouTube/SourceFromYouTube.tsx [hunk 2]
```diff
@@ -47,7 +49,7 @@ export function SourceFromYouTube({
             overflow: 'hidden'
           }}
         >
-          YouTube &nbsp;
+          {t('Youtube \u00A0')}
```
Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/VideoBlockEditor/Source/SourceFromYouTube/SourceFromYouTube.tsx:52]:
```suggestion
          {t('YouTube \u00A0')}
```

File: apps/journeys-admin/src/components/Editor/ActionsTable/ActionsList/ActionsListView/ActionsListView.tsx [hunk 3]
```diff
@@ -66,11 +69,13 @@ export function ActionsListView({
...
               <TableCell>
-                <Typography variant="subtitle2">Target and Action</Typography>
+                <Typography variant="subtitle2">
+                  {t('Target and Action')}
+                </Typography>
               </TableCell>
               <TableCell>
                 <Typography variant="subtitle2" align="center" width={100}>
-                  Appears on{' '}
+                  {t('Appears on {{space}}', { space: ' ' })}
```
Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/ActionsTable/ActionsList/ActionsListView/ActionsListView.tsx:78]: I dont actually think we need this space
```suggestion
                  {t('Appears on)}
```

File: apps/journeys-admin/src/components/Editor/ActionsTable/ActionsList/ActionsListView/ActionsListView.tsx [hunk 4]
```diff
@@ -185,9 +192,9 @@ export function ActionsListView({
                       </Typography>
                     </Stack>
                     <Typography variant="body2" color="secondary.light">
-                      Appears on{' '}
+                      {t('Appears on {{space}}', { space: ' ' })}
```
Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/ActionsTable/ActionsList/ActionsListView/ActionsListView.tsx:195]:
```suggestion
                      {t('Appears on)}
```

File: apps/journeys-admin/src/components/Editor/ActionsTable/ActionsBanner/ActionsBanner.tsx [hunk 5]
```diff
@@ -117,9 +119,9 @@ export function ActionsBanner(): ReactElement {
               }
             }}
           >
-            On this screen you will see all your goals and actions listed in a
-            single table. Create cards with some actions like buttons. We will
-            list all your links and actions here so you can:
```
Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/ActionsTable/ActionsBanner/ActionsBanner.tsx:122]: Are you able to preserve the number of lines here?

File: apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCardMenu/TrashJourneyDialog/TrashJourneyDialog.tsx [hunk 6]
```diff
@@ -78,9 +80,9 @@ export function TrashJourneyDialog({
       testId="TrashJourneyDialog"
     >
       <Typography>
-        By selecting “delete”, this journey will be moved to the trash. It will
-        remain there for 40 days, before being automatically and permanently
-        deleted.
```
Comment (csiyang) [inline apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCardMenu/TrashJourneyDialog/TrashJourneyDialog.tsx:83]: Preserve number of line

File: apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCardMenu/DeleteJourneyDialog/DeleteJourneyDialog.tsx [hunk 7]
```diff
@@ -78,8 +80,9 @@ export function DeleteJourneyDialog({
       testId="DeleteJourneyDialog"
     >
       <Typography>
-        Are you sure you would like to delete the journey immediately? You will
-        not be able to undo or restore these journeys.
```
Comment (csiyang) [inline apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCardMenu/DeleteJourneyDialog/DeleteJourneyDialog.tsx:82]: Preserve number of lines

**Review by csiyang — CHANGES_REQUESTED** [review]

**Review by csiyang — CHANGES_REQUESTED** [review]
1. On `ActionsTable.tsx` and `ReportsNavigation.tsx` the project used for the translation is wrong, can you please update them also to be `apps-journeys-admin`
2. You still have lint and test errors on the branch, some of which may get fixed by running the extract translations command `nx extract-translations journeys-admin`

File: apps/journeys-admin/src/libs/TestEditorState/TestEditorState.tsx (hunk 1 above)
Comment (zwsaqx) [reply apps/journeys-admin/src/libs/TestEditorState/TestEditorState.tsx:18]: This initializes the translation function t() that was required to resolve lint errors for the text on lines 22-51
